### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by kjaymiller**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,72 @@
...
+![api_key in buttondown-settings](assets/api_buttondown-settings.jpg)
+
+- `GITHUB_ACCOUNT` - Your github account name. This is the name of the account that owns the repo you want to pull issues from.
+- `GITHUB_REPO` - The name of the repo you want to pull issues from.
+
+### Optional Values
+
+- `GITHUB_API_TOKEN` - Your github api token. You can create one in your GitHub account settings in the Developer Settings section. If you have one, you can use it to increase the rate limit of the GitHub API. You can read more about the GitHub API rate limit [here](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting).
+    
+![Github Access Token](assets/personal_access_token.png)
```
Comment (jonafato) [inline README.md]: Can you make this alt text ("GitHub Access Token") more descriptive of the image here?

File: README.md [hunk 2]
```diff
@@ -1,35 +1,72 @@
...
+```bash
+python publish_show.py 1 \ 
+--buttondown_api_key=<your_api_key> \ 
+--github_account=<your_github_account> \
+--github_repo=<your_github_repo> \
+--github_api=<your_github_api_token>
+```
+
+You can also call the script with the `--help` flag to see all the options.
+![](assets/publish_show_help.png)
```
Comment (jonafato) [inline README.md]: I'm guessing you want the color formatting in the image you've included here, but can you also provide a text version directly in the readme and add alt text to the image?

**Review by jonafato — DISMISSED** [review]
LGTM, just two requests regarding the images included in the readme.

File: README.md (hunk 2 above)
Comment (kjaymiller) [reply README.md]: Alt Text absolutely!

The text version of the output may not format correctly, but I can try.

File: README.md (hunk 1 above)
Comment (kjaymiller) [reply README.md]: This image was removed to make the variables more connected

File: README.md (hunk 2 above)
Comment (kjaymiller) [reply README.md]: Yeah the output was a garbled mess that said I did add the full command python `publish_newsletter.py --help`

**Review by jonafato — APPROVED** [review]

**Pull request merged**